lib/Runtime/Library/JavascriptSIMDBool32x4.cpp

```diff
@@ -97,10 +97,10 @@ namespace Js
         JavascriptSIMDBool32x4 *instance = JavascriptSIMDBool32x4::FromVar(args[0]);
         Assert(instance);
 
-        wchar_t stringBuffer[1024];
+        wchar_t stringBuffer[SIMD_STRING_BUFFER_MAX];
         SIMDValue value = instance->GetValue();
 
-        swprintf_s(stringBuffer, 1024, L"SIMD.Bool32x4(%s, %s, %s, %s)", value.i32[SIMD_X] ? L"true" : L"false", value.i32[SIMD_Y] ? L"true" : L"false", value.i32[SIMD_Z] ? L"true" : L"false", value.i32[SIMD_W] ? L"true" : L"false");
+        ToStringBuffer(value, stringBuffer, SIMD_STRING_BUFFER_MAX);
```

Comment:
Can we use JavascriptSIMDBool32x4::ToStringBuffer to be consistent with usage of static functions